torch/quantization/fx/quantize.py

```diff
@@ -621,15 +613,16 @@ def get_qconfig(module):
             elif node.op == 'call_module':
                 self.qconfig_map[node.name] = get_qconfig(self.modules[node.target])
 
-    def _prepare(self, model, qconfig_dict, inplace, quant_type):
+    def _prepare(self, model, qconfig_dict, inplace, is_dynamic):
+        assert not inplace, 'inplace prepare is not supported yet'
         input_root = model.root
         if not inplace:
             input_root = copy.deepcopy(input_root)
 
         input_graph = model.graph
-        self.quant_type = quant_type
+        self.is_dynamic = is_dynamic
```

Comment:
why not keep the enum?  it's more readable

Reply:
it looks like we don't need to distinguish between static and qat. they follow the same flows